workspaceConfig.json

```diff
@@ -0,0 +1 @@
+{"sources":[],"metadata":{"sourceListStore":{"sources":[],"firstLoad":false},"destinationListStore":{"destinations":[],"firstLoad":false},"connectionsStore":{"connections":{}},"version":"1.0.1"}}
```

Comment:
Use empty file?

Reply:
added the above because it gives an idea to the user, what file it is if the file name was confusing.

Reply:
maybe call it workspaceConfig.json.example?